app/src/main/java/com/nutomic/syncthingandroid/activities/WebGuiActivity.java

```diff
@@ -177,4 +198,63 @@ private void loadCaCert() {
             }
         }
     }
+
+    /**
+     * Set webview proxy and sites that are not retrieved using proxy.
+     * Compatible with KitKat or higher android version.
+     * Returns boolean if successful.
+     * Source: https://stackoverflow.com/a/26781539
+     */
+    public static boolean setWebViewProxy(Context appContext, String host, int port, String exclusionList) {
+        Properties properties = System.getProperties();
+        properties.setProperty("http.proxyHost", host);
+        properties.setProperty("http.proxyPort", port + "");
+        properties.setProperty("https.proxyHost", host);
+        properties.setProperty("https.proxyPort", port + "");
+        properties.setProperty("http.nonProxyHosts", exclusionList);
+        properties.setProperty("https.nonProxyHosts", exclusionList);
+
+        if (android.os.Build.VERSION.SDK_INT < Build.VERSION_CODES.KITKAT) {
+            // Not supported on android version lower than KitKat.
+            return false;
+        }
+
```

Comment:
This can be above meddling with the properties, or potentially just skip the alternative reload